packages/manager/apps/pci-instances/src/components/menu/ActionsMenu.component.tsx

```diff
@@ -10,57 +7,24 @@ import {
   Divider,
   Icon,
   ICON_NAME,
-  Link,
   Popover,
   PopoverContent,
   PopoverTrigger,
 } from '@ovhcloud/ods-react';
 import { DeepReadonly } from '@/types/utils.type';
-
-export type TActionsMenuItem = DeepReadonly<{
-  label: string;
-  link: {
-    path: string;
-    isExternal: boolean;
-  };
-}>;
+import {
+  ActionMenuItem,
+  TActionsMenuItemProps,
+} from './ActionMenuItem.component';
 
 export type TActionsMenuProps = DeepReadonly<{
-  items: Map<string, TActionsMenuItem[]>;
+  items: Map<string, TActionsMenuItemProps[]>;
```

Comment:
Should be agnostic from props notation as Map's values are filled with items (`TActionsMenuItem[]`)

Reply:
Yes, but what I'm saying is that an `ActionsMenuItem` is a model for the view, so callers are building a Map with entities, not props that are very coupled to the react component

Reply:
done